core-web/libs/template-builder/src/lib/template-builder.module.ts

```diff
@@ -36,7 +37,8 @@ import { TemplateBuilderComponent } from './components/template-builder/template
         DotTemplateBuilderStore,
         DialogService,
         DynamicDialogRef,
-        DotAddStyleClassesDialogStore
+        DotAddStyleClassesDialogStore,
+        DotContainersService
```

Comment:
Looks like this provider was necessary if we wanted the service to be available for the directive when the webapp is running.